DigitalLearningSolutions.Web/ViewModels/TrackingSystem/EditCentreWebsiteDetailsViewModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models;
+
+    public class EditCentreWebsiteDetailsViewModel
+    {
+        public EditCentreWebsiteDetailsViewModel() { }
+
+        public EditCentreWebsiteDetailsViewModel(Centre centre)
+        {
+            CentreTelephone = centre.CentreTelephone;
+            CentreEmail = centre.CentreEmail;
+            CentrePostcode = centre.CentrePostcode;
+            OpeningHours = centre.OpeningHours;
+            CentreWebAddress = centre.CentreWebAddress;
+            OrganisationsCovered = centre.OrganisationsCovered;
+            TrainingVenues = centre.TrainingVenues;
+            OtherInformation = centre.OtherInformation;
+        }
+
+        [MaxLength(100, ErrorMessage = "Enter a telephone number that is no more than 100 characters.")]
+        public string? CentreTelephone { get; set; }
+
+        [EmailAddress(ErrorMessage = "Enter a valid email address.")]
+        [MaxLength(100, ErrorMessage = "Enter an email address that is no more than 100 characters.")]
+        public string? CentreEmail { get; set; }
+
+        [Required(ErrorMessage = "Enter a postcode.")]
+        [MaxLength(10, ErrorMessage = "Enter a postcode that is no more than 10 characters.")]
+        public string? CentrePostcode { get; set; }
+
+        [MaxLength(100, ErrorMessage = "Enter no more than 100 characters for the opening hours.")]
+        public string? OpeningHours { get; set; }
+
+        [MaxLength(100, ErrorMessage = "Enter a web address that is no more than 100 characters.")]
+        public string? CentreWebAddress { get; set; }

```

Comment:
Do we need any validation for obviously non-web addresses?

Reply:
Just checked and there's nothing done in the old system, so I think it's ok to keep no validation as the scope. 👍